pages/manage/productions/create.vue

```diff
@@ -53,16 +53,14 @@
           />
         </section>
         <section v-else>
-          <b-form-group :label="$t('productions.create.production_name')">
-            <b-form-radio
-              v-for="production in availableProductions"
-              :key="production.id"
-              v-model="selectedSuggestedProductionId"
-              :value="production.id"
-              name="production"
-              >{{ production.title }}</b-form-radio
-            >
-          </b-form-group>
+          <pub-radio-group
+            v-model="selectedSuggestedProductionId"
+            :group-label="$t('productions.create.production_name')"
+            :values="availableProductionIds"
+            :labels="availableProductionTitles"
```

Comment:
Can't we use collection here (array with objects)? 
That would make it more clear when your values and labels don't align.

Reply:
Done